src/js/playground/playground.js

```diff
@@ -1,7 +1,7 @@
 import Blockly from "blockly";
 import "../catblocks_msgs";
 import "./../blocks";
-import { checkNextBlock, wrapElement } from '../share/utils';
+import {checkNextBlock, wrapElement} from '../share/utils';
```

Comment:
as far as I have seen, we always have these spaces in imports

Reply:
added whitespaces againg.